ParisBNF/abb/BNFabb20.xml

```diff
@@ -52,60 +93,212 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <material key="parchment"/>
                         </support>
                                 <extent>
-                                    <measure unit="leaf">120</measure>
+                                    <measure unit="leaf">128</measure>
                                     <dimensions type="outer" unit="mm">
                                         <height>290</height>
                                         <width>270</width>
                                     </dimensions>
                                 </extent>
+                                
+                                <foliation>Foliation in the upper right corner of the leaves.</foliation>
+                                <collation>The quire structure cannot be ascertained on the basis of the available images.</collation>
+                                <condition key="intact"/>
                             </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="18 19">
+                                    <locus from="3r" to="127v"/>                                   
+                                    <dimensions type="outer" unit="mm">
+                                        <height>275</height>
+                                        <width>257</width>                                        
+                                    </dimensions>
+                                  
+                                    <ab type="ruling"/>
+                                    <ab type="pricking">Ruling is visible.</ab>
+                                    <ab type="ruling">Pricking is visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C.</ab>                                                                       
+                                    <note>The leaves <locus target="#1 #2 #128"/> are not ruled.</note>
+                                </layout>
+                            </layoutDesc>
                         </objectDesc>
+                        <handDesc>
+                            
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <locus from="3ra" to="125vb"/>
+                                <locus from="126rb_l19" to="128rc"/>
+                                <seg type="script">End of the 18th/beginning of the 18th century according to C. Conti Rossini, and 18th/19th century according to 
```

Comment:
"End of the 18th/beginning of the **19th** century according to C. Conti Rossini,"

Reply:
ok